data/json/items/ranged.json

```diff
@@ -595,10 +595,12 @@
     "aim_speed" : 4,
     "recoil": 60,
     "durability": 9,
-    "clip_size": 20,
-    "reload": 400,
+    "reload": 75,
     "loudness" : 125,
-    "valid_mod_locations": [[ "accessories", 2 ],[ "barrel", 1 ],[ "bore", 1 ],[ "conversion", 1 ],[ "grip", 1 ],[ "magazine", 2 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
+    "magazines" : [ "lp_9mm_mag" ],
+    "default_mods": [ "lp_9mm_mag" ],
```

Comment:
The further PR I'm working on won't support this so it might be better to remove the `default_mods` line for now so we don't have two competing systems when magazine spawning is worked out